.github/workflows/confrimations-summary.yml

```diff
@@ -0,0 +1,29 @@
+name: Confirmations summary
+on:
+  push:
+    paths:
+      - ui/pages/confirmations/**
+  pull_request:
+    branches:
+      - develop
+    paths:
+      - ui/pages/confirmations/**
+
+jobs:
+  comment:
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
+      contents: write
+    steps:
+      - uses: actions/github-script@v7
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          script: |
+            github.rest.issues.createComment({
+              issue_number: context.issue.number,
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              body: ':warning: The confirmations are currently being redesigned. You have made changes to `ui/pages/confirmations/**`. Contact the confirmations team before finalizing your changes. :warning:'
```

Comment:
I still think we should call out explicitly where the old flows and the new flows are located, and that any changes to one should also be ported to the other.